src/System.Globalization/tests/DateTimeFormatInfo/DateTimeFormatInfoGetAbbreviatedDayName.cs

```diff
@@ -16,8 +16,7 @@ public static IEnumerable<object[]> GetAbbreviatedDayName_TestData()
             yield return new object[] { new CultureInfo("en-US").DateTimeFormat, englishAbbreviatedDayNames };
             yield return new object[] { new DateTimeFormatInfo(), englishAbbreviatedDayNames };
 
-            // ActiveIssue(2103)
-            if (!PlatformDetection.IsUbuntu1510 && !PlatformDetection.IsUbuntu1604 && !PlatformDetection.IsUbuntu1610)
+            if (!PlatformDetection.IsUbuntu || PlatformDetection.IsUbuntu1404)
```

Comment:
we still support Ubuntu 14.04 so I think we need to keep it till we drop the support